src-docs/charm.py.md

```diff
@@ -18,7 +18,11 @@ Charm for Synapse on kubernetes.
 ## <kbd>class</kbd> `SynapseCharm`
 Charm the service. 
 
+<<<<<<< HEAD
```

Comment:
left-over from merge

Reply:
Thanks, for some reason I lost my pre-commit config :| Now is fixed.